Minor training updates (#66)
inmanta/examples: Training/0-manual-configuration.md

# Manual configuration

> Despite already being deployed, the routers and subscribers are still not connected to each other. We can atest this using `ping`. Here we try to ping from subscriber-east to router-east:

Reviewer: ```suggestion
Despite already being deployed, the routers and subscribers are still not connected to each other. We can test this using `ping`. Here we try to ping from subscriber-east to router-east:
```

## Configure the network manually - Subscribers

1. Configure the interface manually.  The subscribers containers are simple linux hosts, as for most linux hosts, we can simply use the `ip` command.

> So on subscriber east we would execute this command.

Reviewer: ```suggestion
On subscriber east we will execute the following command:
```
